### DISCUSSION TITLE
Prisma and jsonapi specification for ExpressJS

### DISCUSSION THREAD

**Discussion by aspyropoulos** [prisma/prisma#13890] [Q&A]
I want to build an json server using Prisma and [jsonapi](https://jsonapi.org/) specification. Is there any know package that combines these two for ExpressJS?

**Comment by austincrim** [comment]
Hey @aspyropoulos 👋,

I'm not aware of any package that combines these technologies together.

Let us know about your experience, though!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
